lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -400,6 +400,8 @@ void IMConstPropPass::markBlockExecutable(Block *block) {
     else if (auto cast = dyn_cast<mlir::UnrealizedConversionCastOp>(op))
       for (auto result : cast.getResults())
         markOverdefined(result);
+    else if (auto verbatim = dyn_cast<VerbatimExprOp>(op))
+      return markOverdefined(verbatim.getResult());
```

Comment:
I think we cannot `return` here

Reply:
Thanks @uenoku , very careless copy paste bug.

Reply:
Yeah, I also have done this kind of a mistake so many times 😅